GATEWAY/src/pages/applications/applications.ts

```diff
@@ -28,7 +28,17 @@ export class ApplicationsPage {
               public modalCtrl: ModalController,
               private mqttClient: MqttClient,
               private tilesApi: TilesApi,
-              private storage: Storage) {}
+              private storage: Storage,
+              private plt: Platform,
+              private alertCtrl: AlertController) {
+
+    if (this.plt.is('android')) {
```

Comment:
Won't this always present if the user has android? Even if the location is enabled?